src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const textData = document.querySelectorAll('.population');
```

Comment:
Consider using a more descriptive variable name for 'textData', such as 'populationElements'.